openwisp_notifications/admin.py

```diff
@@ -1,16 +1,62 @@
 from django.contrib import admin
+from openwisp_notifications.base.admin import NotificationSettingAdminMixin
 from openwisp_notifications.swapper import load_model
 
 from openwisp_users.admin import UserAdmin
 from openwisp_utils.admin import AlwaysHasChangedMixin
 
 Notification = load_model('Notification')
-NotificationUser = load_model('NotificationUser')
+NotificationSetting = load_model('NotificationSetting')
 
 
-class NotificationUserInline(AlwaysHasChangedMixin, admin.StackedInline):
-    model = NotificationUser
-    fields = ['receive', 'email']
+@admin.register(NotificationSetting)
+class NotificationSettingAdmin(
+    NotificationSettingAdminMixin, admin.ModelAdmin,
```

Comment:
why did you add this? What can this do that we can't already do from the user admin?

Reply:
Operators can not view `User` admin, for them I had to add this. 
This also simplifies editing notification settings by superuser as they can filter via notification types and organization and edit notification settings for users accordingly.  We can hide this for `Superusers` and `Administrators` if you like.

Reply:
I want to avoid having to add more items to the admin index page, because on installations which use many modules it becomes huge, at some point all those links will have to go in the menu because the index will give way to a general dashboard, but also the menu cannot be huge or users will be overwhelmed.

I suggest the following:

- we can just remove this now
- we can open an issue to add a dedicated admin view to allow the current logged in user to change their notification settings, we can then add a link like "change notification preferences" somewhere in the notification JS widget, this would be great for UX